src/mint/library.cairo

```diff
@@ -199,6 +199,31 @@ namespace CarbonableMinter:
         return (TRUE)
     end
 
+    func withdraw{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
```

Comment:
I think I'll implement this proposition if you agree (which handles all cases I think):

```cairo
    func withdraw() -> (success : felt):
    end

    func transfer(token_address : felt, to : felt, amount : Uint256) -> (success : felt):
    end
```

> Open question: should we add the amount in parameter as well or always withdraw all available balance

I think `amount` is required since 2 users could have sent both the same wrong token, so admin can directly send tokens and the right amount to specific users through the minter smart contract.

Reply:
Done in [907b787](https://github.com/Carbonable/carbonable-starknet/pull/21/commits/907b7878ae24719431625fb7573a82364d550b5a)

Note: no test added to specific `transfer`  function since `withdraw` calls the new function under the hood.